README.rst

```diff
@@ -234,13 +234,34 @@ Configuration:
     # The maximum size of files to load into DataStore. In bytes. Default is 1 GB.
     ckanext.xloader.max_content_length = 1000000000
 
-    # To always use messytables to load data, instead of attempting a direct
-    # PostgreSQL COPY, set this to True. This more closely matches the
-    # DataPusher's behavior. It has the advantage that the column types
+    # By default, xloader will first try to add tabular data to the DataStore
+    # with a direct PostgreSQL COPY. This is relatively fast, but does not
+    # guess column types. If this fails, xloader falls back to a method more
+    # like DataPusher's behaviour. This has the advantage that the column types
     # are guessed. However it is more error prone, far slower and you can't run
     # the CPU-intensive queue on a separate machine.
```

Comment:
You totally can run the queue on a separate machine. data.qld.gov.au has been doing it for years. You just need access to the Redis instance that stores the jobs, as well as the database, and a copy of the CKAN configuration file.

Reply:
Makes sense. I think I will just remove the statement that you can't run it separately, instead of trying to convey the necessities of doing so in this limited space.